packages/config/src/evalConfig.ts

```diff
@@ -22,23 +21,43 @@ export function evalConfig(
   configFile: string,
   request: ConfigContext | null
 ): DynamicConfigResults {
-  const babel = require('@babel/core');
+  const contents = readFileSync(configFile, 'utf8');
+  try {
+    const { code } = transform(contents, {
+      filePath: configFile,
+      transforms: ['typescript', 'imports'],
+    });
 
-  const { code } = babel.transformFileSync(configFile, {
-    // only: [configFile],
-    cwd: request?.projectRoot || process.cwd(),
-    babelrc: false,
-    configFile: false,
-    comments: false,
-    ignore: [/node_modules/],
-    filename: 'unknown',
-    presets: [getBabelPreset()],
-    // Retain lines for improved error reporting.
-    retainLines: true,
-  });
+    const result = requireString(code, configFile);
+    return resolveConfigExport(result, configFile, request);
```

Comment:
Move these out of the try-catch since presumably we don't want to apply code frame formatting to errors in these method calls. (Keep try-blocks as narrow as possible so that they catch errors only from the code we expect to throw an error.)

Reply:
`requireString` can throw syntax errors, other one could be fine to move out.